west.yml

```diff
@@ -195,7 +195,7 @@ manifest:
       groups:
         - hal
     - name: hal_microchip
-      revision: 32a79d481c056b2204a5701d5a5799f9e5130dd7
```

Comment:
remove the `PR #31` from the commit title

Reply:
Done